app-modules/student-data-model/src/Providers/StudentDataModelServiceProvider.php

```diff
@@ -67,6 +71,12 @@ public function boot(): void
             'student_email_address' => StudentEmailAddress::class,
             'sms_opt_out_phone_number' => SmsOptOutPhoneNumber::class,
             'enrollment_semester' => EnrollmentSemester::class,
+            'bounced_email_address' => BouncedEmailAddress::class,
         ]);
+
+        Event::listen(
+            SesBounceEvent::class,
+            SaveBouncedEmailAddress::class
+        );
```

Comment:
Do we feel strongly that this belong here and not `IntegrationAwsSesEventHandlingServiceProvider`?

Given than it may not be for a Student, but a Prospect, and doesn't necessarily have to do with the Student directly.

Reply:
Yeah, I personally would leave it tbh, even if it isn't optimal for prospects. We did use the student module for general "educatable" files in the past